iroha/src/kura.rs

```diff
@@ -90,6 +90,20 @@ impl Kura {
             }
         }
     }
+
+    pub fn latest_block_hash(&self) -> Hash {
```

Comment:
`last_block`?

Reply:
`Last` has the meaning of being final it would mean that no more blocks would be added. So I think `latest` is good to use here.